test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2575,6 +2575,23 @@ def forward(self, x):
         x = torch.randn(2, 3, 4)
         self.run_test(TensorFactory(), x)
 
+    @skipIfUnsupportedMinOpsetVersion(9)
+    def test_eye(self):
+        class TensorFactory(torch.nn.Module):
+            def forward(self, x):
+                return torch.eye(x.size()[1], 3), torch.eye(4, 4, dtype=torch.long), torch.eye(x.size()[1], 2, dtype=torch.long)
+
+        x = torch.randn(2, 3, 4)
+        self.run_test(TensorFactory(), x)
+
+        class TensorFactoryScript(torch.jit.ScriptModule):
```

Comment:
To test this dynamic feature, we should first export a model with m,n=4, then test it with different values, such as m,n=5, otherwise, it's hard to say whether correctly support this dynamic feature or just encoded the static values.

Reply:
makes sense, updated with to test with input of different shape